blockchain/contracts/HelloWorld.sol

```diff
@@ -25,15 +27,18 @@ contract HelloWorld {
         NFTOwnership memory nftOwnership;
         nftOwnership.nftId = nftHash;
         nftOwnership.owner = nftOwner;
+        nftOwnership.droptime = 1637085033;
         nftOwnerships[nftHash] = nftOwnership;
         nfts.push(nftHash);
         return nftHash;
     }
 
-    function buyNFT(uint256 price, uint256 nftHash) public payable {
+    function buyNFT(uint256 price, uint256 nftHash) public payable returns(address) {
+        require(nftOwnerships[nftHash].droptime<=block.timestamp,string(abi.encode("Drop has not started yet! ",(nftOwnerships[nftHash].droptime-block.timestamp),"Seconds left!")));
         price = msg.value;
         nftOwner.transfer(price);
         nftOwnerships[nftHash].owner = buyer;
+        return nftOwnerships[nftHash].owner;
```

Comment:
Why did you add this return value?

Reply:
Thought it would be beneficial for testing but turns out its not, resolved